src/Bicep.Core/Parser/Lexer.cs

```diff
@@ -248,7 +248,11 @@ private void LexToken()
             
             if (tokenType == TokenType.Unrecognized)
             {
-                AddDiagnostic(b => b.UnrecognizedToken(tokenText));
+                if (tokenText == "\"") {
+                    AddDiagnostic(b => b.DoubleQuoteToken());
+                } else {
```

Comment:
nit - `} else {` is a very TS/JS-y styling. Can you match the rest of the formatting?